src/cart/src/cart/crt_group.h

```diff
@@ -256,7 +254,7 @@ struct crt_lookup_item {
 	hg_addr_t		 li_tag_addr[CRT_SRV_CONTEXT_NUM];
 
 	/* reference count */
-	uint32_t		 li_ref;
+	ATOMIC uint32_t		 li_ref;
 	uint32_t		 li_initialized:1;
 	pthread_mutex_t		 li_mutex;
```

Comment:
And this one?

Reply:
The same as above. Probably this code can be also elaborated but in separate patch.

Reply:
@dmiter could you please fix that in a subsequent patch?

Reply:
Ok. I will look how to do this.